plugin/client_rpc.go

```diff
@@ -99,6 +99,9 @@ func init() {
 	gob.Register(&model.AppError{})
 	gob.Register(&ErrorString{})
 	gob.Register(&opengraph.OpenGraph{})
+	gob.Register(&model.AutocompleteDynamicListArg{})
+	gob.Register(&model.AutocompleteStaticListArg{})
+	gob.Register(&model.AutocompleteTextArg{})
```

Comment:
Should `AutocompleteData` also be added to the list?

Reply:
No, this is needed only for objects which are provided using `interface{}` type